sdks/python/apache_beam/transforms/combiners_test.py

```diff
@@ -485,6 +520,113 @@ def test_fixed_windows_combine(self):
                   equal_to([('c', 3), ('c', 10), ('d', 5), ('d', 17)]),
                   label='sum per key')
 
+  # Test that three different kinds of metrics work with a customized
+  # CounterIncrememtingCombineFn.
+  def test_custormized_counters_in_combine_fn(self):
+    p = TestPipeline()
+    input = (p
+             | beam.Create([('c', 'i'),
+                            ('c', 'go'),
+                            ('c', 'run'),
+                            ('d', 'beam'),
+                            ('d', 'tests')]))
+
+    # The result of concatenating all values regardless of key.
+    global_concat = (input
+                     | beam.Values()
+                     | beam.CombineGlobally(CounterIncrememtingCombineFn())
+                     | "sort global result" >> _SortLists)
```

Comment:
There's no need for _SortLists anywhere here.

Reply:
Deleted.